torchvision/prototype/datasets/utils/_resource.py

```diff
@@ -162,10 +167,12 @@ def resolve(self) -> OnlineResource:
             for attr in (
                 "file_name",
                 "sha256",
-                "_preprocess",
-                "_loader",
```

Comment:
Remove _loader because it cause error and as of now we can't see why it needed here (as self._loader is a function defined in the parent class)